dashboard/modules/node/tests/test_node.py

```diff
@@ -104,6 +104,7 @@ def getpid(self):
             detail = detail["data"]["detail"]
             assert detail["hostname"] == hostname
             assert detail["raylet"]["state"] == "ALIVE"
+            assert detail["raylet"]["isHeadNode"] == True
```

Comment:
Is it possible to check if this is False when it is a worker node?

Reply:
it's not easy to test since we're using ip to determine head node. Do you know a better way to determine if a node is the head node?

Reply:
Hmm I see. Let's leave it as it is now. We will add a new flag I think. We can address it after that